api/orchestrator/orchestrator.proto

```diff
@@ -184,6 +184,14 @@ service Orchestrator {
     };
   }
 
+  // Retrieves target cloud service statistics
+  rpc CloudServiceStatistics(CloudServiceStatisticsRequest)
```

Comment:
rename to `GetCloudServiceStatistics`?

Reply:
See answer below. Maybe we should use GetStatistics() and move it to orchestrator.go?